README.md

```diff
@@ -1,44 +1,112 @@
-# Plarform Libraries
+# Platform Libraries
 
-## License
+A monorepo containing 20+ PHP libraries for the Keboola platform. All libraries are located in the `libs/` directory. Each library is independently versioned and published but developed together for easier testing and cross-library changes.
+
+## Libraries
+
+### Data Movement
+- **input-mapping** - Downloads tables/files from Storage API (CSV/Parquet support)
+- **output-mapping** - Uploads data to Storage API with import handling
+- **staging-provider** - Configures staging factories for local/workspace storage
+- **slicer** - Data slicing utilities
+
+### API Clients
+- **api-bundle** - Symfony bundle for Keboola API applications with authentication
+- **service-client** - Generic service client for Keboola services
+- **query-service-api-client** - Query Service API client
+- **sync-actions-api-php-client** - Sync Actions API client
+- **sandboxes-service-api-client** - Sandboxes Service API client
+- **azure-api-client** - Azure API client
+- **vault-api-client** - Vault API client
+
+### Infrastructure
+- **k8s-client** - Kubernetes client wrapper
+- **messenger-bundle** - Symfony Messenger bundle with Azure Service Bus support
+- **doctrine-retry-bundle** - Database retry logic for Doctrine
+- **logging-bundle** - Logging configuration bundle
+
+### Utilities
+- **configuration-variables-resolver** - Resolves configuration variables
+- **permission-checker** - Permission checking utilities
+- **key-generator** - Key generation utilities
+- **settle** - Settlement utilities for async operations
+- **php-test-utils** - Shared testing utilities
 
-MIT licensed, see [LICENSE](./LICENSE) file.
+## Development
 
-## CI setup
+### Environment Setup
 
-### Auto Stop for Azure Synapse Analytics
```

Comment:
We are not executing CI tests for Synpase backend. Synapse backend is not longer supported by KBC